packages/mobile/src/identity/reducer.ts

```diff
@@ -236,6 +236,18 @@ export const reducer = (
           false
         ),
       }
+    case Actions.DELETE_USER_DATA:
+      return {
```

Comment:
Are these correct? I'm not sure what they all do so some of these may be unnecessary and I might be missing some other important ones, please take a close look :)

The idea here is to save stuff that is not directly linked to the account being deleted and would save time/computation when a new account is registered on the device.

Reply:
I'm not 100% clear on this, but I understand that some of these mappings involve making requests that use (a lot of?) bandwidth and if we already did that for people on the phone's contact list it might make sense to keep them since we'll likely need to do them again anyways if the user logs in again.
@jmrossy is that correct?

Reply:
There's some value in saving them but the potential edge cases and buts they raise may not be worth the value in keeping them. The most valuable would be `e164NumberToSalt`. Can you start a slack thread for us to chat this.
We definitely don't need `importContactsProgress` and `askedContactsPermission` but the others _may_ be worth saving.